### PR TITLE
Small improvements to the docs

### PR DESCRIPTION
**Pull request by KobeW50**
Idk how to name PRs so i just give them the most generic name possible 🙈

To see how the callouts look, see [this](https://github.com/orgs/community/discussions/16925)

### CODE REVIEW

**Comment by KobeW50** [comment]
Also, just curious: 

I noticed the ARSClib cli uses the `-a` flag now for the input apk, even tho previously it was just added at the end of the command without a flag.  Also the `-o` flag now seems to only specify the output directory and not the filename. 

1. Will the standard cli also get these changes?

2. With the ARSClib cli, is there a way to name the output apk (instead of `base.apk`)?

File: docs/latest-reddit-patch-info.md [hunk 1]
```diff
@@ -13,26 +13,28 @@ Currently, only the CLI method is available.
...
+- The option '-e' allows you to exclude patches. (e.g. `-e "Disable screenshot popup"` ...)
 
-2. Run the CLI
 ```
 java -jar revanced-cli-all.jar \
- -a reddit.apk \
- -o revanced \
- -m app-release-unsigned.apk \
+ -a input.apk \
+ -o output \
```
Comment (KobeW50) [inline docs/latest-reddit-patch-info.md]:
```suggestion
 -o output-folder \
```

File: docs/revancify.md [hunk 2]
```diff
@@ -60,11 +65,13 @@ From the Version Selection Menu, choose the `Auto Select` option or a version la
 
 The version will be downloaded and patching will begin.
 
+> [!TIP]
```
Comment (KobeW50) [inline docs/revancify.md]:
```suggestion
> [!NOTE]
```

**Comment by inotia00** [comment]
> 1. Will the standard cli also get these changes?

No these changes only apply to ARSClib cli

> 2\. With the ARSClib cli, is there a way to name the output apk (instead of `base.apk`)?

No user can only change the output directory name, and the name of the output apk file cannot be changed from `base.apk` to a different name

**Comment by KobeW50** [comment]
Thanks. Btw this is ready for review

**Comment by inotia00** [comment]
Thanks

**Pull request merged**